index.html

```diff
@@ -12611,6 +12630,24 @@ <h3>IDL Attribute Name Notes or Exceptions</h3>
 		</section>
 	</section>
 
+	<section id="idl_element">
+		<h2><code>AccessibilityMixin</code> Mixed in to <code>Element</code></h2>
+
+		<p>User agents must include <code>AccessibilityMixin</code> on <code>Element</code>:</p>
```

Comment:
Do you want lowercase must or uppercase MUST?

Reply:
I'm not sure what this group's convention is; editorial help is welcome. For other web specs we prefer lowercase, per https://infra.spec.whatwg.org/#conformance, but perhaps things are different here.

Reply:
For RFC2119 as stated at https://w3c.github.io/aria/#conformance we require uppercase MUST.